custom-snippets/giveGreetingChallenge/test.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+submission_file=${1:-submission.txt}
+
+cp "$submission_file" submission.js
```

Comment:
this falls victim to the hard-to-decipher syntax error when there's no forced blank after the file and if the student didn't terminate with a ;

```suggestion
cp "$submission_file" submission.js
echo >> submission.js
```